tests/e2e/specs/auth-flow-proxy.test.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * WordPress dependencies
+ */
+import { deactivatePlugin, visitAdminPage } from '@wordpress/e2e-test-utils';
+
+/**
+ * Internal dependencies
+ */
+import { resetSiteKit } from '../utils';
+
+describe( 'Site Kit set up flow for the first time', () => {
+	afterEach( async () => {
+		await resetSiteKit();
+	} );
+
+	it( 'renders a splash page for proxy set up when no GCP credentials are provided', async () => {
+		await deactivatePlugin( 'e2e-tests-gcp-credentials-plugin' );
```

Comment:
This isn't necessary since `deactivateUtilityPlugins` is called in `beforeAll` and `afterAll` in the bootstrap.
```suggestion
```